libslide/src/bignum/modulo.rs

```diff
@@ -36,6 +40,9 @@ fn single_skip_mod(lhs: Bignum, rhs: Bignum) -> Bignum {
     if !rhs.dec.is_empty() {
         panic!("The module fuction only supports whole numbers!");
```

Comment:
```suggestion
        panic!("The modulo fuction only supports whole numbers!");
```

can you fix this real quick